addon/rollup.config.mjs

```diff
@@ -0,0 +1,40 @@
+import ts from 'rollup-plugin-ts';
```

Comment:
@vstefanovic97 could you please latest addon-blueprint where we migrated away from `rollup-plugin-ts`?